pages/common/ajson.md

```diff
@@ -9,7 +9,7 @@
 
 - Read JSON from `stdin` and execute a specified JSONPath expression:
 
-`cat {{path/to/file.json}} | ajson '{{$..json[?(@.path)]}}'`
+`ajson '{{$..json[?(@.path)]}}' < {{path/to/file.json}}`
```

Comment:
```suggestion
`ajson < {{path/to/file.json}} '{{$..json[?(@.path)]}}'`
```
While this is not a direct approval of the idea, I have an improvement suggestion: Always have the shell redirection come after the program invocation

Reply:
```suggestion
`ajson <{{path/to/file.json}} '{{$..json[?(@.path)]}}'`
```
Also I wonder if having no space in between makes the example more or less clear to the user viewing the example.

Reply:
I guess it's not something that can be used universally. `ip rule` for example has writing to a file and reading from a file on the same spot which makes it easy to read.